### PR TITLE
[velero] feat: add nodeselector for restic

### PR DESCRIPTION
**Pull request by peterekhator**
Signed-off-by: Peter Ekhator <peterekhator@gmail.com>

#### Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [x] [DCO](https://github.com/helm/charts/blob/master/CONTRIBUTING.md#sign-your-work) signed
- [x] Chart Version bumped
- [x] Variables are documented in the values.yaml or README.md
- [x] Title of the PR starts with chart name (e.g. `[velero]`)

### CODE REVIEW

File: charts/velero/templates/restic-daemonset.yaml [hunk 1]
```diff
@@ -152,4 +152,10 @@ spec:
       tolerations:
         {{- toYaml . | nindent 8 }}
       {{- end }}
+      {{- if .Values.restic.nodeSelector }}
+      nodeSelector:
+        {{- with .Values.restic.nodeSelector }}
+          {{- toYaml . | nindent 8 }}
+        {{- end }}
+      {{- end }}
```
Comment (jenting) [inline charts/velero/templates/restic-daemonset.yaml:160]: Suggestion change
```yaml
      {{- with .Values.restic.nodeSelector }}
      nodeSelector:
        {{- toYaml . | nindent 8 }}
      {{- end }}
```

**Review by jenting — CHANGES_REQUESTED** [review]

File: charts/velero/templates/restic-daemonset.yaml (hunk 1 above)
Comment (peterekhator) [reply charts/velero/templates/restic-daemonset.yaml]: thanks @jenting. pushed an update

**Review by jenting — DISMISSED** [review]
Thank you 🚀

**Comment by jenting** [comment]
@carlisia @cpanato PTAL

**Review by cpanato — APPROVED** [review]

**Review by jenting — APPROVED** [review]

**Pull request merged**